Gemfile

```diff
@@ -21,13 +21,18 @@ gem 'jquery-rails'
 
 # Use datatables for rendering and searching songs
 gem 'jquery-datatables-rails', '~> 3.3.0'
+gem 'will_paginate'
 
+gem 'yomu'
+gem 'tesseract-ocr'
```

Comment:
Oh that makes sense. Wow...I didn't know some PDFs were just scans. If I knew that, I probably wouldn't have so casually assigned this script without doing more research myself.